java/dataset/src/main/java/org/apache/arrow/dataset/substrait/AceroSubstraitConsumer.java

```diff
@@ -96,15 +96,17 @@ private ArrowReader execute(String plan, Map<String, ArrowReader> namedTables) t
         ArrowArrayStream streamOutput = ArrowArrayStream.allocateNew(this.allocator)
     ) {
       String[] mapTableToMemoryAddress = getMapTableToMemoryAddress(namedTables, arrowArrayStream);
-      JniWrapper.get().executeSerializedPlanNamedTables(
+      JniWrapper.get().executeSerializedPlan(
           plan,
           mapTableToMemoryAddress,
           streamOutput.memoryAddress()
       );
       return Data.importArrayStream(this.allocator, streamOutput);
     } finally {
-      for (ArrowArrayStream stream : arrowArrayStream) {
-        stream.close();
+      try {
+        AutoCloseables.close(arrowArrayStream);
+      } catch (Exception e) {
+        throw new RuntimeException(e);
```

Comment:
This method is declared to throw Exception already. Why are we catching and re-throwing this?

Reply:
Why are we catching it in the first place? Let it propagate.

Reply:
ok, changed